### PR TITLE
Add VIRTUAL_CACHE layer to fix Cache server icon in topology

### LINKED ISSUE #9404: [Bug] v9.1.0 Redis in Topogy always not show as technology icon but a question mark ,
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

In Skywalking Java , Lettuc or Jedis Plugin span is SpanLayer.Cache;
but in OAP, when OAP analysis from SpanLayer to layer (RPCAnalysisListener#identifyRemoteServiceLayer), it will be Layer.CACHE(9,true);
because of Filed "isNormal" is true,  when gen ServiceID  ,"isReal" will be true, so graphQL result will be null. 
Redis Node won't  show as his technology icon

### What you expected to happen

maybe add a VIRTUAL_CACHE like VIRTUAL_DATABASE to Layer 

### How to reproduce

start up OAP v.9.1.0 release version ,  and use java-agent v8.11.0 , java app use lettuce connect to redis, 
will see question mark in topology , redis node.

### Anything else

..

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by StreamLang**
↓ ↓ topology screenshot ↓ ↓  
before : 

before

after : 

after

### Fix `https://github.com/apache/skywalking/issues/9404`
- [x] Add a unit test to verify that the fix works.
- [x] Explain briefly why the bug exists and how to fix it. 
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #9404.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

**Comment by wu-sheng** [comment]
Please update changes.md in the PR template. You missed one step.
And remove <> around the issue number.

**Comment by wu-sheng** [comment]
General good. wait for polish.

**Comment by StreamLang** [comment]
> Please update changes.md in the PR template. You missed one step. And remove <> around the issue number.

please check again.

**Review by wu-sheng — APPROVED** [review]
Thanks for fixing. LGTM.

File: docs/en/changes/changes.md [hunk 1]
```diff
@@ -28,6 +28,7 @@
 * Fix race condition in Banyandb storage
 * Support `SUM_PER_MIN` downsampling in `MAL`.
 * Support `sumHistogramPercentile` in `MAL`.
+* Add `VIRTUAL_CACHE` to Layer, to fix [#9404](https://github.com/apache/skywalking/issues/9404)
```
Comment (wu-sheng) [inline docs/en/changes/changes.md:31]:
```suggestion
* Add `VIRTUAL_CACHE` to Layer, to fix conjectured Redis server, which icon can't show on the topology.

**Review by wu-sheng — APPROVED** [review]
Adjust change log a little to make things clear

**Pull request merged**